README.md

```diff
@@ -12,7 +12,7 @@ This project assumes our standard tech stack for frontend projects:
 
 - [Gulp](https://gulpjs.com) for running tasks [1]
 - [Rollup](https://rollupjs.org/guide/en/) for building JavaScript bundles
-- [Sass](https://sass-lang.com) for authoring styles
+- [Tailwind](https://tailwindcss.com) or [Sass](https://sass-lang.com) for authoring styles
```

Comment:
Use of SASS is now optional, and projects should generally use either Tailwind or SASS for each CSS bundle. Using both is possible, but more annoying to configure.